travis-fix-coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+cat > .coveragerc <<EOF
+[paths]
+source =
+    .
+    $PWD
+EOF
+mv .coverage .coverage.merge
```

Comment:
Is `.coverage` the right name? `mv` complains that there is no `.coverage`: https://travis-ci.org/linux-system-roles/network/jobs/545422641#L238